src/vivarium/framework/results/context.py

```diff
@@ -138,14 +159,13 @@ def _format(field, param):
             return f"{str(field).upper()}_{param}"
 
         for params, val in data.iteritems():
+            if isinstance(params, str):  # handle single stratification case
+                params = [params]
             key = "_".join(
                 [_format("measure", measure)]
-                + [_format(field, measure) for field, param in zip(data.index.names, params)]
+                + [_format(field, param) for field, param in zip(data.index.names, params)]
                 # Sorts additional_keys by the field name.
-                + [
-                    _format(field, measure)
-                    for field, param in sorted(additional_keys.items())
-                ]
+                + [_format(field, param) for field, param in sorted(additional_keys.items())]
             )
             results[key] = val
```

Comment:
I found this section a bit confusing but the takeaway is it is parsing the output data and building the results table?

Reply:
It's formatting the string for the specific stratification (e.g., `MEASURE_person_time_SEX_female_AGE_25_to_30_year_2045`)